gatling/simulations/LoadTest.scala

```diff
@@ -0,0 +1,36 @@
+
+import scala.concurrent.duration._
+import java.util.concurrent.TimeUnit
+
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+import io.gatling.jdbc.Predef._
+
+class LoadTest extends Simulation {
+
+	val httpProtocol = http
+		.baseURL("http://hippo-delivery.web.dev.ps.nhsd.io")
```

Comment:
Is it worth pulling this out as a configuration param, even if just as a local const to highlight that it may want to be changed?